### PR TITLE
Minor revisions

### PR DESCRIPTION
**Pull request by ghost**
For normative text:
- Update abstract
- Clarify "final" means following AliasMode indirection
- Specify that TargetName CNAME target base domain must be resolved first
- Clarify that SNI and base name match the actual TLSA record found

For examples:
- State that DNSSEC is assumed for examples
- Add "URI" when describing example services for clarity
- Fix 0 priority in ServiceMode record
- Explain transport label is selected by client
- Explain CNAME fallback sequence
- Minor formatting

### CODE REVIEW

File: draft-rebs-dnsop-svcb-dane.md [hunk 1]
```diff
@@ -56,15 +56,17 @@ This draft describes the interaction of DANE with indirection via Service Bindin
 
 This draft applies the same logic to SVCB-compatible records.  Specifically, if SVCB resolution was entirely secure (including any AliasMode records and/or CNAMEs), then for each connection attempt derived from a SVCB-compatible record,
 
-* The TLSA base domain MUST be the final SVCB TargetName.
+* The TLSA base domain MUST be the final SVCB TargetName after following AliasMode indirection if applicable.
```
Comment (bemasc) [inline draft-rebs-dnsop-svcb-dane.md:59]:
```suggestion
* The initial TLSA base domain MUST be the final SVCB TargetName used for this connection attempt.  (Names appearing earlier in a resolution chain are not used.)
```

File: draft-rebs-dnsop-svcb-dane.md [hunk 2]
```diff
@@ -56,15 +56,17 @@ This draft describes the interaction of DANE with indirection via Service Bindin
...
 * The transport prefix MUST be the transport of this connection attempt (possibly influenced by the "alpn" SvcParam).
 * The port prefix MUST be the port number of this connection attempt (possibly influenced by the "port" SvcParam).
 
-If the SVCB TargetName contains a CNAME record, clients MUST try to use both the alias target and owner as the base domain, as described in {{Section 7 of RFC7671}}.  If the TLSA QNAME is aliased by a CNAME, clients MUST follow the CNAME to complete the resolution.  (This does not alter the TLSA base domain.)
+If the SVCB TargetName contains a CNAME record, clients MUST first try to use the CNAME target as the TLSA base domain then fallback to the final SVCB TargetName as the TLSA base domain, as described in {{Section 7 of RFC7671}}.
```
Comment (bemasc) [inline draft-rebs-dnsop-svcb-dane.md:63]:
```suggestion
If the SVCB TargetName contains a CNAME record, clients MUST first try to use the CNAME target as the TLSA base domain, with fallback to the final SVCB TargetName as the TLSA base domain, as described in {{Section 7 of RFC7671}}.
```

File: draft-rebs-dnsop-svcb-dane.md [hunk 3]
```diff
@@ -128,69 +134,74 @@ The TLSA QNAME is `_443._tcp.api.example.com`.
...
 The TLSA QNAME is `_443._tcp.xyz.example-cdn.com`.
 
 ## QUIC and CNAME
 
-Given service https://api.example.com and records:
+Given service URI `https://api.example.com` and records:
 
 ~~~
 www.example.com.  CNAME api.example.com.
 api.example.com.  HTTPS 1 svc4.example.net alpn=h3 port=8443
```
Comment (bemasc) [inline draft-rebs-dnsop-svcb-dane.md:153]:
```suggestion
api.example.com.  HTTPS 1 svc4.example.net alpn=h2,h3 port=8443
```

File: draft-rebs-dnsop-svcb-dane.md [hunk 4]
```diff
@@ -128,69 +134,74 @@ The TLSA QNAME is `_443._tcp.api.example.com`.
...
 svc4.example.net. CNAME xyz.example-cdn.com.
 ~~~
 
+If the client supports `alpn=h3`, the transport label is set to `_quic` otherwise `_tcp` is used.
+
 The TLSA QNAME would be one of:
```
Comment (bemasc) [inline draft-rebs-dnsop-svcb-dane.md:159]:
```suggestion
The initial TLSA QNAME would be one of:
```

File: draft-rebs-dnsop-svcb-dane.md [hunk 5]
```diff
@@ -128,69 +134,74 @@ The TLSA QNAME is `_443._tcp.api.example.com`.
...
 
-Given service foo://api.example.com:8443 and record:
+Given service URI `foo://api.example.com:8443` and record:
 
 ~~~
 _8443._foo.api.example.com. SVCB 1 api.example.com.
 ~~~
 
-The TLSA QNAME is `_8443._$PROTO.api.example.com`, where $PROTO is the appropriate value as discussed in {{protocols}}.
+The TLSA QNAME is `_8443._$PROTO.api.example.com`, where $PROTO is the appropriate value as discussed in {{protocols}} matching the client-selected transport.
```
Comment (bemasc) [inline draft-rebs-dnsop-svcb-dane.md:177]:
```suggestion
The TLSA QNAME is `_8443._$PROTO.api.example.com`, where $PROTO is the appropriate value for the client-selected transport as discussed in {{protocols}} .
```

File: draft-rebs-dnsop-svcb-dane.md [hunk 6]
```diff
@@ -128,69 +134,74 @@ The TLSA QNAME is `_443._tcp.api.example.com`.
 
 ## HTTPS AliasMode
 
-Given service https://api.example.com and records:
+Given service URI `https://api.example.com` and records:
 
 ~~~
 api.example.com.     HTTPS 0 svc4.example.net.
-svc4.example.net.    HTTPS 0 xyz.example-cdn.com.
+svc4.example.net.    HTTPS 1 xyz.example-cdn.com.
```
Comment (bemasc) [inline draft-rebs-dnsop-svcb-dane.md:141]: I liked this better the other way.  HTTPS does allow the chain to terminate with an AliasMode record.  This way seems too similar to the previous example.

Comment (evansrg) [reply draft-rebs-dnsop-svcb-dane.md]: Reverted

**Comment by evansrg** [comment]
Updated and squashed

**Review by bemasc — APPROVED** [review]

**Pull request merged**